Makefile

```diff
@@ -132,21 +136,13 @@ logging-container: demo-container ## Run dashboard server with node debugger in
 
 ############### Unit and integration tests
 
-test-container: CMD?=test
-test-container: SERVER_NAME?=$(TEST_CONTAINER_NAME)
-test-container: DOCKER_OPTIONS?=
-test-container:
-	@docker run -it --rm \
-		--name $(SERVER_NAME) \
-		$(DOCKER_OPTIONS) \
-		$(TEST_IMAGE_NAME) $(CMD)
-
-test: | build test-container ## Run unit tests
-
-IT_SERVER_NAME:=integration-test-server
+test: ## Run unit tests
+	docker run -it --rm \
+		--name $(TEST_CONTAINER_NAME) \
+		$(TEST_IMAGE_NAME) test
```

Comment:
Missing `@`.

Also, I get `_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.` as output, but everything seems to work fine.